api/custom_auth/serializers.py

```diff
@@ -18,17 +20,30 @@ class Meta:
 
 
 class CustomUserCreateSerializer(UserCreateSerializer):
+    key = serializers.SerializerMethodField()
+
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
```

Comment:
I don't think this is doing anything anymore? We can get rid of the whole `__init__` method.